From d690cf1ee8f61eb86a9bd6f21aa5aea8a7b4d2a3 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sat, 22 Sep 2012 17:44:32 -0400
Subject: [PATCH] Fix python build by using libxsltmod_la_CPPFLAGS instead of
 AM_CPPFLAGS

Commit a2cd8a03 broke the --with-python build by moving includes from
INCLUDES to AM_CPPFLAGS. AM_CPPFLAGS gets ignored when a target-specific
*_CPPFLAGS variable exists, but at least some automake versions
automatically add "libxsltmod_la_CPPFLAGS = -shared" to python/Makefile.in

https://bugzilla.gnome.org/show_bug.cgi?id=684637
---
 python/Makefile.am | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/python/Makefile.am b/python/Makefile.am
index 48a6c6e..6869c2b 100644
--- a/python/Makefile.am
+++ b/python/Makefile.am
@@ -3,12 +3,6 @@ AUTOMAKE_OPTIONS = 1.4 foreign
 
 SUBDIRS= . tests
 
-AM_CPPFLAGS = \
-	-I$(PYTHON_INCLUDES) \
-	-I$(top_srcdir)/libxslt \
-	-I$(top_srcdir) \
-	-I../libexslt
-
 AM_CFLAGS = $(LIBXML_CFLAGS)
 
 DOCS_DIR = $(datadir)/doc/libxslt-python-$(LIBXSLT_VERSION)
@@ -36,6 +30,11 @@ all-local: libxslt.py
 
 python_LTLIBRARIES = libxsltmod.la
 
+libxsltmod_la_CPPFLAGS += \
+	-I$(PYTHON_INCLUDES) \
+	-I$(top_srcdir)/libxslt \
+	-I$(top_srcdir) \
+	-I../libexslt
 libxsltmod_la_SOURCES = libxslt.c types.c
 nodist_libxsltmod_la_SOURCES = libxslt-py.c
 libxsltmod_la_LIBADD = $(mylibs) $(PYTHON_LIBS)
-- 
1.7.12

